spyderlib/plugins/ipythonconsole.py

```diff
@@ -177,12 +173,22 @@ def setup_page(self):
 
         interface_layout = QVBoxLayout()
         interface_layout.addWidget(banner_box)
-        interface_layout.addWidget(gui_comp_box)
         interface_layout.addWidget(pager_box)
         interface_layout.addWidget(calltips_box)
         interface_layout.addWidget(ask_box)
         interface_group.setLayout(interface_layout)
 
+        comp_group = QGroupBox(_("Completion Type"))
+        comp_label = QLabel(_("Decide what type of completion to use"))
+        comp_label.setWordWrap(True)
+        completers = [("Graphical", 0), ("Terminal", 1), ("Plain", 2)]
+        comp_box = self.create_combobox(_("Completion:")+"   ", completers,
+                                        'completion_type', default=0)

```

Comment:
Please remove the `default` value. This is deprecated and it's going to be removed in 3.0.

Sorry I forgot to tell you before :-)